code/modules/antagonists/pirate/pirate_gangs.dm

```diff
@@ -98,6 +102,7 @@ GLOBAL_LIST_INIT(pirate_gangs, init_pirate_gangs())
 
 	threat_title = "Transfer of goods"
 	threat_content = "Ahoy! This be the %SHIPNAME. Cough up %PAYOFF credits or you'll walk the plank."
+	arrival_announcement = "Jolly Roger won't wait forever, maties; we're by the starboard ready to send you some round gifts."
```

Comment:
```suggestion
	arrival_announcement = "The Jolly Roger won't wait forever, maties; we're laying alongside ready to send you some gifts."
```

"laying alongside" is a nautical term for when two vessels are adjacent to each other, "starboard" doesn't really work here because the ship could actually be on the port, thus making the announcement quite silly.
"Jolly Roger" is a proper noun, referring to the ship.